docs/about/cedar-design-system/README.md

```diff
@@ -14,45 +14,71 @@
+###  Design tokens
+Design tokens are special variables used to maintain a scalable visual system for UI development and brand consistency. Cedar design tokens store the visual design attributes that define the foundation of REI’s visual language, including color, typography, and spacing.
 
-New feature and enhancement requests can be submitted on [#cedar-user-support](https://rei.slack.com/messages/CA58YCGN4) or during our weekly office hours.
+Learn more about design tokens in our [design tokens overview](../../tokens/overview/).
 
-###  Report an Issue
-The Cedar team is committed to maintaining consistent, high-quality components. If you encounter any defects in the Figma UI Toolkit, Vue code, or on the documentation site, please submit them to [#cedar-user-support](https://rei.slack.com/messages/CA58YCGN4) or come and talk to us during our weekly office hours.
+###  Components
+Components use Cedar’s tokens and design foundations to distribute code templates that extend basic HTML elements and encapsulate reusable code. These custom markup elements represent specific portions of the user interface. When used in your application, they will help ensure the UI remains consistent with REI’s digital display standards.
+  
+For example:
+`<cdr-link href=”rei.com”>CdrLink Component</cdr-link>`
 
-The team will address the bug as soon as possible, depending on the severity of the issue and the sprint capacity. Please include as much relevant information as possible when submitting a bug report, including steps to reproduce, urgency, and impact to help prioritize the task.
+### Component variables
+  
+Component variables are available for a subset of Cedar components. These SCSS or LESS mixins extend design tokens and define additional specific variables that are used to display Cedar components.
+  
+For example:
+`cdr-link-base-mixin` would be used to style an element like a CdrLink component.
 
-If you have a fix for the bug and would like to submit a PR for it, please reach out in Slack at [#cedar-user-support](https://rei.slack.com/messages/CA58YCGN4)  or visit the [Contributing to Cedar](../../about/contributing-to-cedar/) page to learn more about contribution opportunities.
+Learn more in our article on [component variables](../../components/component-variables/).
+  
+### Patterns
+  
+Patterns are reusable frameworks providing guidance to common design problems at REI. Patterns may be UI or a variety of other things, like a behavior, flow, or an application motif. For example, the alert pattern gives design and development guidance for various alert scenarios, while the form input types pattern offers common form types like street address.
+  
+### Documentation site
+  
+Documentation for each component and foundational style includes design guidelines, do’s and don’ts, considerations for accessibility, and API instructions for developers. This is the site you’re on right now—bookmark it for easy reference while working.
 
```

Comment:
Would we want to include anything here about the code repositories? I think of the system being 1. the design toolkit 2. code to match the designs and 3. the doc site that brings it all together. Those are made up of 1. Design tokens, 2. components and component variables, 3. patterns

Reply:
I like this idea. Would the order be:

- Design toolkit
- Code repositories
- Documentation site
- Design tokens
- Components
- Component variables
- Patterns

@mhewson do you have any definitions of the code repositories on hand? Or a rough idea of the language needed, and I can fine-tune it